test/test_basic.py

```diff
@@ -80,30 +84,43 @@ def request(self, url, retries=0):
         self.fail(f"timeout reached trying to request {url}")
 
     def testBasic(self):
-        response = self.request('https://localhost:8443/api/webconsole/v1/sessions/new')
+        response = self.request(f'{URL}{config.ROUTE_CONTROL}/sessions/new')
         self.assertEqual(response.status, 200)
         sessionid = json.load(response)['id']
         self.assertIsInstance(sessionid, str)
 
         podman = ['podman', 'run', '-d', '--pod', 'webconsoleapp',
                   '--network', 'consoledot', 'localhost/webconsoleserver']
         cmd = ['websocat', '--basic-auth', 'admin:foobar', '-b', '-k',
-               f'wss://host.containers.internal:8443/wss/webconsole-ws/v1/sessions/{sessionid}',
+               f'wss://host.containers.internal:8443{config.ROUTE_HOST}/sessions/{sessionid}',
```

Comment:
Hmm this is left out? Even thought it is internal, it would not be served on 8443 but 443 in k8s. So hmm this could maybe actually become 443 as everything should be bound in the same network NS

Reply:
Good point -- conceptually it should use the publicly visible URL, and "host.containers.internal" is the equivalent to "localhost". So maybe that is trying to tell us to actually only move the port number into a variable, and leave the https://localhost bit as-is?

Reply:
Reworked as discussed, to set the port in the Makefile.